.github/workflows/build-LaTex.yaml

```diff
@@ -0,0 +1,14 @@
+name: "Build LaTeX"
+
+on: push
```

Comment:
Tienes que poner los paths para que solo lo haga cuando cambien los ficheros tex, no todos los ficheros. Por favor, mira la documentación de GitHub actions: https://docs.github.com/es/actions/using-workflows/events-that-trigger-workflows
Como os he dicho muchas veces, no se trata de que hagáis cosas, se trata de que aprendáis a hacer cosas.